examples/hosts/app-integration/external-controller/src/controller.ts

```diff
@@ -33,8 +38,13 @@ const diceValueKey = "diceValue";
  * The DiceRoller is our data object that implements the IDiceRoller interface.
  */
 export class DiceRollerController extends EventEmitter implements IDiceRollerController {
-    constructor(private readonly kvPairDataObject: IKeyValueDataObject) {
+    constructor(
+        private readonly fluidContainer: FluidContainer,
```

Comment:
The controller doesn't need the container (here it's just wrapping and reexposing properties and events).  Let's just pass the container straight into the view rather than having the controller act as a middleman?  Maybe best as two separate view functions (`renderDiceRoller(diceRoller, div)` and `renderAudience(FluidContainer, div)`)?  This will also demonstrate a reusable approach for non-dice-roller scenarios.